### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by dubstard**
<body>

Site | URLScan link
-- | --
apibridgenetwork.com|https://urlscan.io/result/63f19a84-931b-4564-acf3-78baad10c8e8/
auth-webconnect.net|https://urlscan.io/result/766ea84e-0d28-4d04-8d2c-cccc9ef7b26c/dom/
mainnetconnect-wallets.com|https://urlscan.io/result/18331e29-7459-4a48-a58a-0df06cf60df0/dom/
crovalidates.live|https://urlscan.io/result/e459a4e2-2b96-4701-8ca3-41c32a1fb04b/dom/
corriectionbot­.co|https://user-images.githubusercontent.com/49607867/163664714-a5360ecd-022e-4438-892e-07e6a5c8552d.png
nftwalletclaims.com|https://user-images.githubusercontent.com/49607867/163665221-f8043368-e784-4316-b839-477878c4d1e8.png

</body>
</html>

`apibridgenetwork.com`
image

`auth-webconnect.net `
image

`crovalidates.live`
image

`mainnetconnect-wallets.com`
image

`corriectionbot­.co`
image

`nftwalletclaims.com`
image
```
                "accountvalidation.pages.dev",
		"coinfactory.digital",
                "corriectionbot­.co",
		"cryptolightinvestment.com",
		"eth-airdrops.io",
		"finras.info",
		"foodmoodnft.io",
		"laundapps-connect.us",
		"linksafe-secure.com",
		"livemints.events",
		"mainnetconnect-wallets.com",
		"metamask-cn.win",
		"metamaskregister.com",
		"metamaskwebs.net",
		"metamaskwebs.phantomsapp.io",
		"mindblowon.livemints.events",
		"mint-moodoctopus.io",
		"multiversepad.net",
		"nftwalletclaims.com",
		"nftwalletconnect.finras.info",
		"olympusdao-finance.online",
		"opensea-nft-wallet.com",
		"phantomsapp.io",
		"polkastarter-breeder.netlify.app",
		"stepn.plus",
		"tokenswap.cf",
		"trustbots.support",
		"tw-hm.tel",
		"uniswap-pos.top",
		"walet-polygone.online",
```

### CODE REVIEW

**Review by dubstard — COMMENTED** [review]
2 are duplicates and need to go
  --   "app.multiversepad.net",
  --   "mints-nft.live",

**Review by dubstard — COMMENTED** [review]
add "corriectionbot.co",

**Comment by dubstard** [comment]
✅Ready for review

File: all.json [hunk 1]
```diff
@@ -15,6 +15,44 @@
 		"1inch-wallet.org",
 		"1inch.tk",
 		"1metamask.com",
+		"accountvalidation.pages.dev",
+		"azukis.art",
+		"bayc-land.mints-nft.live",
+		"bnbfaucet.site",
+		"bonowatch.com",
```
Comment (jacogr) [inline all.json:22]: Not sure about this one with regards to crypto?

File: all.json [hunk 2]
```diff
@@ -15,6 +15,44 @@
...
+		"coinfactory.digital",
+		"comicsapesocialsclub.com",
```
Comment (jacogr) [inline all.json:24]: This looks like just an NFT?

File: all.json [hunk 3]
```diff
@@ -15,6 +15,44 @@
 		"1inch-wallet.org",
 		"1inch.tk",
 		"1metamask.com",
+		"accountvalidation.pages.dev",
+		"azukis.art",
+		"bayc-land.mints-nft.live",
```
Comment (jacogr) [inline all.json:20]: Blocked already?

File: all.json [hunk 4]
```diff
@@ -15,6 +15,44 @@
...
+		"bnbfaucet.site",
+		"bonowatch.com",
+		"coinfactory.digital",
+		"comicsapesocialsclub.com",
+		"corriectionbot.co",
+		"cryptolightinvestment.com",
+		"eth-airdrops.io",
+		"faucetlists.ga",
```
Comment (jacogr) [inline all.json:28]: This one just seems to list faucets? Clicked through to a couple and they are external and _seems_ legit?

File: all.json [hunk 5]
```diff
@@ -15,6 +15,44 @@
...
+		"foodmoodnft.io",
+		"laundapps-connect.us",
+		"linksafe-secure.com",
+		"livemints.events",
+		"mainnetconnect-wallets.com",
+		"metamask-cn.win",
+		"metamaskregister.com",
+		"metamaskwebs.net",
+		"metamaskwebs.phantomsapp.io",
+		"mindblowon.livemints.events",
```
Comment (jacogr) [inline all.json:39]: Not just a normal NFT? (the above main site falls into the same category)

File: all.json (hunk 1 above)
Comment (dubstard) [reply all.json:22]: all those are fake mint sites when you try to connect your wallet it gets drained

File: all.json (hunk 5 above)
Comment (dubstard) [reply all.json:39]: Fake NFT mint.
Any interaction with it result in devastating consequences for whoever was naive enough to let their web wallet interact with it.
Many also have fake Connect wallet seed stealing sub pages.
Gonna remov em .

File: all.json (hunk 4 above)
Comment (dubstard) [reply all.json:25]: the .ga threw me off. freenom.
very little legit freenom sites overall (close to below 1%)

**Review by dubstard — COMMENTED** [review]
removed false positives 
"comicsapesocialsclub.com",
"faucetlists.ga",

File: all.json (hunk 1 above)
Comment (dubstard) [reply all.json]: removed them

File: all.json (hunk 2 above)
Comment (dubstard) [reply all.json]: comicsapesocialsclub.com appears to be legit, not fake mint, removed

**Review by dubstard — COMMENTED** [review]
add "apibridgenetwork.com",
removed "bayc-land.mints-nft.live", as the main domain is already blocked anyway

File: all.json (hunk 3 above)
Comment (dubstard) [reply all.json]: removed the subdomain entry, as the main domain was already blocked, my bad (i opened it on a browser without the polka extension)

File: all.json (hunk 1 above)
Comment (dubstard) [reply all.json]: [bonowatch.com/docu/mail0/semtex.php](https://urlscan.io/result/7ef72149-1e35-4cf9-87fb-0d2618b2c3ca/)

https://any.run/report/9f9a3d1a4209be0ce04503d1a806330acb3a400b078ffc17235baf0d5c9701b1/29e55100-85ff-4420-a31a-69c8826aed5e

**Comment by dubstard** [comment]
Should be good now 👍

**Review by ical10 — APPROVED** [review]

**Comment by dubstard** [comment]
thanks

**Pull request merged**